skewer.yaml

```diff
@@ -105,36 +106,57 @@ steps:
               configuration:
                 brokers:
                   - broker: 0
-                    advertisedHost: cluster1-kafka-0.cluster1-kafka-brokers
+                    advertisedHost: cluster1-kafka-brokers
       ~~~
 
       See [Advertised addresses for brokers][advertised-addresses] for
       more information.
 
       [advertised-addresses]: https://strimzi.io/docs/operators/in-development/configuring.html#property-listener-config-broker-reference
-  - standard: kubernetes/create_your_sites
-  - standard: kubernetes/link_your_sites
+  - title: Create your sites
+    preamble: |
+      A Skupper _site_ is a location where components of your
+      application are running.  Sites are linked together to form a
+      network for your application.  In Kubernetes, a site is associated
+      with a namespace.
+
+      Use the kubectl apply command to declaratively create sites in the kubernetes
+      namespaces. This deploys the Skupper router. Then use kubectl get site to see
+      the outcome.
+
+      **Note:** If you are using Minikube, you need to [start minikube
+      tunnel][minikube-tunnel] before you run `skupper init`.
+
+      [minikube-tunnel]: https://skupper.io/start/minikube.html#running-minikube-tunnel
+    commands:
+      private:
+        - run: kubectl apply -f ./private-crs/site.yaml
+        - run: kubectl wait --for condition=Ready --timeout=60s site/private
+          output: |
+            site.skupper.io/private created
+            site.skupper.io/private condition met
+      public:
+        - run: kubectl apply -f ./public-crs/site.yaml
+        - run: kubectl wait --for condition=Ready --timeout=60s site/public
+          output: |
+            site.skupper.io/public created
+            site.skupper.io/public condition met
+  - standard: skupper/link_your_sites/kubernetes_cli
```

Comment:
Something to ponder is whether using standard is appropriate for linking sites. The sample out put provided is not contextual to the example (e.g. different names, etc.) It may be confusing to users.